python/rascaline/rascaline/calculator_base.py

```diff
@@ -197,100 +197,104 @@ def compute(
                   \frac{\partial \langle q \vert A_i \rangle}
                        {\partial \mathbf{r_j}}
 
               where :math:`\langle q \vert A_i \rangle` is the representation around
-              atom :math:`i` and :math:`\mathbf{r_j}` is the position vector of the
-              atom :math:`j`.
+              atom :math:`i` and :math:`\mathbf{r_j}` is the position vector of the atom
+              :math:`j`.
 
               **Note**: Position gradients of an atom are computed with respect to all
               other atoms within the representation. To recover the force one has to
               accumulate all pairs associated with atom :math:`i`.
 
-            - ``"cell"``, for gradients of the representation with respect to cell
-              vectors. Cell gradients are computed as
+            - ``"strain"``, for gradients of the representation with respect to strain.
+              These gradients are typically used to compute the virial, and from there
+              the pressure acting on a system. To compute them, we pretend that all the
+              positions :math:`\mathbf r` and unit cell :math:`\mathbf H` have been
+              scaled by a strain matrix :math:`\epsilon`:
 
               .. math::
-                  \frac{\partial \langle q \vert A_i \rangle}
-                       {\partial \mathbf{h}}
-
-              where :math:`\mathbf{h}` is the cell matrix.
+                 \mathbf r &\rightarrow \mathbf r \left(\mathbb{1} + \epsilon \right)\\
+                 \mathbf H &\rightarrow \mathbf H \left(\mathbb{1} + \epsilon \right)
 
-              **Note**: When computing the virial, one often needs to evaluate
-              the gradient of the representation with respect to the strain
-              :math:`\epsilon`. To recover the typical expression from the cell
-              gradient one has to multiply the cell gradients with the cell
-              matrix :math:`\mathbf{h}`
+              and then take the gradients of the representation with respect to this
+              matrix:
 
               .. math::
-                  -\frac{\partial \langle q \vert A \rangle}
-                        {\partial\epsilon}
-                   = -\frac{\partial \langle q \vert A \rangle}
-                           {\partial \mathbf{h}} \cdot \mathbf{h}
+                  \frac{\partial \langle q \vert A_i \rangle}
+                       {\partial \mathbf{\epsilon}}
 
-        :param selected_samples: Set of samples on which to run the calculation.
-            Use :py:obj:`None` to run the calculation on all samples in the
-            ``systems`` (this is the default).
+            - ``"cell"``, for gradients of the representation with respect to the
+              system's cell parameters. These gradients are computed at fixed positions,
+              and often not what you want when computing gradients explicitly (they are
+              mainly used in ``rascaline.torch`` to integrate with backward
```

Comment:
Should be maybe mention here again that for stresses one should use gradients with respect to `strain`. I know that this is listed above but just to be sure.